pull_request_template.md

```diff
@@ -2,6 +2,7 @@
 
 - Please keep the title of your pull request short but informative - it will 
   appear in the changelog
+- Please ensure your name is credited on our [Contributors appendix](https://github.com/bids-standard/bids-specification/blob/master/src/99-appendices/01-contributors.md). For assistance, please contact @franklin-feingold or @sappelhoff.
```

Comment:
```suggestion
- Please ensure your name is credited on our [Contributors appendix](https://github.com/bids-standard/bids-specification/blob/master/src/99-appendices/01-contributors.md). To add your name, please edit our [Contributors wiki](https://github.com/bids-standard/bids-specification/wiki/Contributors) and add your name with the type of contribution. For assistance, please contact @franklin-feingold or @sappelhoff.
```

Reply:
![grafik](https://user-images.githubusercontent.com/9084751/63096178-7f530b00-bf6d-11e9-8461-8991ad4498b7.png)
I don't know why, but I don't have permissions to push this :-) ... I know I asked you before, but perhaps this is due to this little tick box on the right side of your PR window on GitHub?  
![grafik](https://user-images.githubusercontent.com/9084751/63096238-b2959a00-bf6d-11e9-801b-c42aea12002a.png)

Any clue what else it could be?

Reply:
yeah the box is ticked. perhaps it is because it is trying to push into my repo rather than a bids-standard one?

Reply:
mmh from my experience with other repos, this shouldn't be a problem if the box is ticked. :man_shrugging: Can you add your own suggestion then?